packer/tools-setup.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+sudo debuginfo-install -q -y kernel
```

Comment:
Does this belong here ? Isn't it better to do this in create-image.sh ? Also I'm wondering what tools use the kernel debug info